locales/ca/messages.json

```diff
@@ -1520,6 +1520,9 @@
     },
     "portsFunction_VTX_MSP": {
         "message": "VTX (MSP)"
+    },
```

Comment:
Please remove. We only need to fill up `locales/en` as changes to other locales will be overwritten.